src/tooltip.tsx

```diff
@@ -569,7 +569,7 @@ export const BaseTooltip: React.FC<BaseTooltipProps> = ({
 
                                                 {onClose && (
                                                     <div className={styles.closeButtonIcon}>
-                                                        <InternalIconButton
+                                                        <IconButton
```

Comment:
Not related. We can use IconButton here, we are not using hasOverlay/isOverMedia in here